k8s/server/overlays/ephemeral/postgres/pg-cluster.yaml

```diff
@@ -1,20 +1,40 @@
+# Ref: https://cloudnative-pg.io/documentation/1.21/recovery/#recovery-from-an-object-store
 apiVersion: postgresql.cnpg.io/v1
 kind: Cluster
 metadata:
   name: cpho-postgres14-cluster
 spec:
   instances: 1
-  resources:
-    requests:
-      cpu: 450m
-      memory: 1926Mi
+  startDelay: 900
+  stopDelay: 900
```

Comment:
Just curious, what are the delay's for?

Reply:
The `stopDelay` is to give a postgres cluster pod time to complete it's operations (mostly related to WAL) in the event of an unexpected / expected pod deletion.

The `startDelay` is to delay probe executions (liveliness / readiness). I think this was configured due to fact that `istio-injection` sometimes takes a while to completed.

Here's the doc reference: https://cloudnative-pg.io/documentation/1.20/instance_manager/#startup-liveness-and-readiness-probes